php/local.ini

```diff
@@ -0,0 +1,2 @@
+upload_max_filesize=10M
+post_max_size=10M
```

Comment:
Since Leaf has no file uploads, we can probably remove this one since defaults work